plugin/pkg/scheduler/scheduler.go

```diff
@@ -46,10 +50,11 @@ type Scheduler struct {
 type Config struct {
 	// It is expected that changes made via SchedulerCache will be observed
 	// by NodeLister and Algorithm.
-	SchedulerCache schedulercache.Cache
-	NodeLister     algorithm.NodeLister
-	Algorithm      algorithm.ScheduleAlgorithm
-	Binder         Binder
+	SchedulerCache      schedulercache.Cache
+	NodeLister          algorithm.NodeLister
+	Algorithm           algorithm.ScheduleAlgorithm
+	Binder              Binder
+	PodConditionUpdater PodConditionUpdater
```

Comment:
Yes - I've seen that comment. I meant adding some comment here, to avoid someone using it inappropriately.